src/components/ReportActionItem/MoneyRequestAction.js

```diff
@@ -123,6 +123,11 @@ const MoneyRequestAction = (props) => {
         }
     };
 
+    const onReportPreviewPressed = () => {
+        Report.openReport(props.requestReportID);
+        Navigation.navigate(ROUTES.getReportRoute(props.requestReportID));
+    };
```

Comment:
The report preview components is not fully used but I think it should have its own callback to make to separate the concerns